assets/i18n/qqq.json

```diff
@@ -86,6 +86,10 @@
 	"I18N_INTERACTIONS_MUSIC_PLAY": "Text displayed in a music interaction. - This text is shown inside a button. When the user clicks the button, the sequences of notes the user has introduced is reproduced.\n{{Identical|Play}}",
 	"I18N_INTERACTIONS_MUSIC_PLAY_SEQUENCE": "Text displayed in a music interaction. - This text is shown inside a button. When the user clicks the button, the sequences of notes the user has to copy is reproduced.",
 	"I18N_INTERACTIONS_MUSIC_SUBMIT": "Text displayed in a music interaction. - The text is shown inside a button. When the user clicks the button, the system checks if the answer is correct and continues to the following action.\n{{Identical|Submit}}",
+	"I18N_INTERACTIONS_PENCILCODEEDITOR_CONFIRMATION_REQUIRED": "Text displayed in pencil code editor interaction - This text is shown when user clicks on the reset code button. This is title of a modal asking for confirmation of user's action",
+    "I18N_INTERACTIONS_PENCILCODEEDITOR_ARE_YOU_SURE_YOU_WANT_TO_RESET_YOUR_CODE?": "This text is displayed in pencil code editor interaction - This text is shown when user clicks on the reset button. This is body of the modal and asks the user whether he is sure to continue with the action",
+    "I18N_INTERACTIONS_PENCILCODEEDITOR_CANCEL": "Text displayed in a pencil code editor interaction - This text is shown when user clicks on reset code button, if user clicks this button in the modal then it cancels user's current action",
+    "I18N_INTERACTIONS_PENCILCODEEDITOR_RESET_CODE": "Text displayed in a pencil code editor interaction - This text is shown when user clicks on reset code button, if user clicks this button, the code editor is reset to it's initial state",
```

Comment:
it's --> its. The general rule is that "it's" is short for "it is", which is not the case here.